### ISSUE TITLE
Not installable on Debian Testing/Sid

### ISSUE THREAD

**Issue by Doc73** [ONLYOFFICE/DesktopEditors#1351]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (DEB package)

### Version information

7.4.1

### Expected Behavior

I would install the app.

### Actual Behavior

The app is not installable due to missing `libgconf-2-4` dependency, no longer available in Debian Testing/Sid

Actually, this bug report is a follow-up of #303

### Reproduction Steps

```
$ sudo apt install ./onlyoffice-desktopeditors_amd64.deb 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'onlyoffice-desktopeditors' instead of './onlyoffice-desktopeditors_amd64.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 onlyoffice-desktopeditors : Depends: libgconf-2-4 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Comment by Doc73** [comment]
Debian Sid with Gnome 44.3

**Comment by Doc73** [comment]
I understand, but consider that these dependencies are obsolete already in Debian Stable, although still available, and are not available anymore in testing (Trixie), the next stable.
I think also that onlyoffice very soon won't be installable in Ubuntu too, due to the same motivation.
BTW, in the meanwhile I'll use appimage (I don't like flatpak)

**Comment by XDmitryK** [comment]
@Doc73 Thanks for the help, I wrote to our developers about this case.
And i created ticked #63804 with your issue

**Comment by Doc73** [comment]
I was able to install latest version of this app in my systems, both in Debian Stable and in Debian Unstable. Please, close this issue and thank you very much!

**Comment by XDmitryK** [comment]
This has been fixed in version 7.5.0 or later.

**Comment by Rita-Bubnova** [comment]
I close this issue. Feel free to comment or reopen it if you got further questions.
